README.md

```diff
@@ -45,3 +45,12 @@ It also has optional support for more formats via external libraries:
 * [TIFF](https://en.wikipedia.org/wiki/Tagged_Image_File_Format) via [libtiff](http://www.libtiff.org/)
 * [WebP](https://en.wikipedia.org/wiki/WebP) via [libwebp](https://developers.google.com/speed/webp/)
 * [XPM](https://en.wikipedia.org/wiki/X_PixMap) via [libXpm](http://xorg.freedesktop.org/)
+
+Besides that, GD depends on some external libraries, which are all optional
+and disabled by default:
+
+* [FreeType](https://freetype.org)
```

Comment:
the entries above tend to explain why the dep is here.  it would help to add a small bit of text explaining the purpose of each one.